docs/reference/modules/cross-cluster-search.asciidoc

```diff
@@ -179,6 +179,82 @@ will be prefixed with their remote cluster name:
 // TESTRESPONSE
 
 [float]
+=== Skipping disconnected clusters
+
+By default all remote clusters that are searched via Cross Cluster Search need to be available when
+the search request is executed, otherwise the whole request fails and no search results are returned
+despite some of the clusters are available. Remote clusters can be made optional through the
+boolean `skip_unavailable` setting, set to `false` by default.
+
+[source,js]
+--------------------------------
+PUT _cluster/settings
+{
+  "persistent": {
+    "search": {
```

Comment:
would it be easier to say `search.remote.cluster_one.skip_unavailable: true`?

Reply:
yep